CHANGELOG.md

```diff
@@ -1,9 +1,14 @@
 ## Changelog
 
-### 4.3.0 (Next)
+### 5.0.0 (Next)
 
+* [#394](https://github.com/alexa-js/alexa-app/pull/394): Upgrade development environment - [@favna](https://github.com/favna).
 * [#388](https://github.com/alexa-js/alexa-app/pull/388): Updating typings for values of AuthorityResolution - [@camerongraybill](https://github.com/camerongraybill).
 
+#### BREAKING CHANGES
+
+* Drop support for all NodeJS versions lower than 10
```

Comment:
No need to shout in CHANGELOG :) Put this in https://github.com/alexa-js/alexa-app/blob/master/UPGRADING.md, business as usual.

Reply:
I'll move it in a few. I put it in changelog because I personally follow angular's changelog template where this is standard. Didn't know you guys want it in upgrading.

Reply:
That's how it was being done until now. We can debate changing that separately I guess, I don't have too strong of an opinion.

Reply:
Moved to upgrading in 71875dc616373455b9ad549665a27078c010d5ec

Reply:
> That's how it was being done until now. 

Nah it's cool, I personally adopted [commitizen](https://github.com/commitizen/cz-cli) and (autogenerated) [conventional changelog](https://github.com/conventional-changelog/conventional-changelog) in my own projects recently but everyone has their own style

Reply:
Concerning this:
> Needs some README changes around compatibility

I changed `4.3.0` to `5.0.0` but I wasn't sure if I should also add a `## Compatibility` block? Not sure to what extend you want to go.